meica.py

```diff
@@ -154,6 +154,13 @@ def getdsname(e_ii,prefixonly=False):
 	if prefixonly: return dsprefix(dsname)
 	else: return dsname
 
+def isnumber(val,ret=False):
```

Comment:
Is there a reason to include the `ret` keyword, here? It's not being used in the main function as far as I can tell, and removing it will simplify the function considerably.